jaeger-crossdock/src/main/java/com/uber/jaeger/crossdock/JerseyServer.java

```diff
@@ -143,12 +149,20 @@ public static void main(String[] args) throws Exception {
     new TChannelServer(tchannelBuilder, new TraceBehavior(), server.getTracer()).start();
   }
 
-  private static Sender senderFromEnv(String jaegerHost) {
+  private static String getEvn(String envName, String defaultValue) {
+    String env = System.getenv(envName);
+    if (env == null) {
+      return defaultValue;
+    }
+    return env;
+  }
+
+  private static Sender senderFromEnv(String collectorHostPort, String agentHostPort) {
```

Comment:
I'm being pedantic here but should s/agentHostPort/agentHost

Reply:
a little bit ;) I will update